service/src/main/kotlin/xyz/lebkuchenfm/Application.kt

```diff
@@ -114,7 +116,20 @@ fun Application.module() {
     val llmSituationsPromptRepository = LlmSituationPromptsMongoRepository(database)
     val llmPromptService = LlmPromptService(llmPersonalityPromptRepository, llmSituationsPromptRepository)
 
-    val textToSpeechProvider = GoogleCloudTextToSpeech(googleCloudTextToSpeechClient)
+    val ttsProviderConfig = environment.config.property("radioPersonality.textToSpeech.ttsProvider").getString()
+    val textToSpeechProvider: TextToSpeechProvider = when (ttsProviderConfig) {
+        "googleCloud" -> {
```

Comment:
maybe we could have switch for that in web client?

Reply:
We should, but right now we cannot! We need a system for changing the configuration during runtime. This is definitely something that's on my mind recently, but it's not really doable in the current version without some stupid hacks.